app/forms/concerns/decidim/reporting_proposals/form_builder_override.rb

```diff
@@ -10,8 +10,11 @@ module FormBuilderOverride
       delegate :asset_pack_path, to: :@template
 
       included do
+        alias_method :original_file_field, :file_field
+
         def file_field(object_name, options = {})
           return super(object_name, options) unless Decidim::ReportingProposals.use_camera_button
+          return original_file_field unless @template.respond_to?(:snippets)
```

Comment:
```suggestion
          return original_file_field(object_name, options) unless Decidim::ReportingProposals.use_camera_button
          return original_file_field(object_name, options) unless @template.respond_to?(:snippets)
```

Now that I see thid, maybe for consistency we should do this?